cdap-support-bundle/bin/collect_support_bundle.py

```diff
@@ -0,0 +1,660 @@
+            "placeholderString": "<svc>",
+            "forEachCommand": f'kubectl describe svc <svc> -n {kube_namespace}',
+            "childFileName": "kubectl-describe-svc",
+            "parentFolder": services_info
+        },
+        {
+            "command": f'kubectl get sts -n {kube_namespace}',
+            "fileName": 'kubectl-get-sts',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": sts_info
+        },
+        {
+            "command": f'kubectl get sts --no-headers -o custom-columns=":metadata.name" -n {kube_namespace}',
+            "fileName": 'kubectl-get-sts',
+            "fileExt": 'log',
+            "fileWrite": False,
+            "forEachNode": True,
+            "placeholderString": "<sts>",
+            "forEachCommand": f'kubectl describe sts <sts> -n {kube_namespace}',
+            "childFileName": "kubectl-describe-sts",
+            "parentFolder": sts_info
+        },
+        {
+            "command": f'kubectl get deployment -n {kube_namespace}',
+            "fileName": 'kubectl-get-deployment',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": deployment_info
+        },
+        {
+            "command": f'kubectl get deployment --no-headers -o custom-columns=":metadata.name" -n {kube_namespace}',
+            "fileName": 'kubectl-get-deployment',
+            "fileExt": 'log',
+            "fileWrite": False,
+            "forEachNode": True,
+            "placeholderString": "<deploy>",
+            "forEachCommand": f'kubectl describe deployment <deploy> -n {kube_namespace}',
+            "childFileName": "kubectl-describe-deployment",
+            "parentFolder": deployment_info
+        },
+        {
+            "command": f'kubectl get configmap -n {kube_namespace}',
+            "fileName": 'kubectl-get-configmap',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": config_map_info
+        },
+        {
+            "command": f'kubectl get configmap --no-headers -o custom-columns=":metadata.name" -n {kube_namespace}',
+            "fileName": 'kubectl-get-configmap',
+            "fileExt": 'log',
+            "fileWrite": False,
+            "forEachNode": True,
+            "placeholderString": "<configmap>",
+            "forEachCommand": f'kubectl describe configmap <configmap> -n {kube_namespace}',
+            "childFileName": "kubectl-describe-configmap",
+            "parentFolder": config_map_info
+        },
+        {
+            "command": f'kubectl logs -l cdap.container.TetheringAgent -n {kube_namespace} --tail -1',
```

Comment:
Sorry, I forgot to mention this earlier - please capture monitoring agent logs from the CDAP system namespace

- kubectl logs test682-monitoring-agent-0 -c monitoring-agent -n {kube_namespace}
- kubectl logs test682-monitoring-agent-0 -c access-token-generator -n {kube_namespace}

Reply:
Both of these commands added with label  control-plane=monitoring-agent